packages/api/src/graphql/mutation.public.ts

```diff
@@ -316,6 +332,8 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
         password: {type: GraphQLNonNull(GraphQLString)},
         passwordRepeated: {type: GraphQLNonNull(GraphQLString)}
       },
+      description:
+        "This mutation allows to update the user's password by entering the new password and the repeated new password and gives an error if the passwords don't match or if the user is not authenticated",
```

Comment:
Maybe better in a split sentence:
"This mutation allows to update the user's password by entering the new password. The repeated new password gives an error if the passwords don't match or if the user is not authenticated."